app/javascript/components/pages/SponsorUs.jsx

```diff
@@ -70,16 +70,13 @@ const SponsorUsWidget = () => {
             <div className="flex flex-col justify-between h-max mt-5 md:mt-0 md:w-auto md:ml-7">
                 {sponsorCardData.map((card) => {
                     return (
-                        <button
-                            key={`${card.type}-options`}
```

Comment:
This button used to be there so that when someone clicked on a sponsorship option, it would switch the main card to reflect that option.

This was actually not accessible, because that functionality should be implemented using just radio buttons. I have removed this button and made changes to the radio button behavior (in another file) to implement it properly.